py/desiperf/instperfapp/data_mgt/data_general.py

```diff
@@ -1,5 +1,6 @@
 
 import pandas as pd
+import fnmatch
```

Comment:
I think you should avoid adding this dependency just to keep the total list of required modules small.

Reply:
Ok, that shouldn't be difficult